### PR TITLE
UI: Clarifying what refers to Sources or Items

### PR DESCRIPTION
**Pull request by ysalmon**
This draft PR is mainly intended to make discussion on https://ideas.obsproject.com/posts/790/add-a-deep-copy-operation-clarify-globality-of-objects-in-the-doc clearer.

### Description

This changes some texts in the UI to explicitly differentiate between Sources and Items.

### Motivation and Context

This is a possible remedy to https://ideas.obsproject.com/posts/790/add-a-deep-copy-operation-clarify-globality-of-objects-in-the-doc
### How Has This Been Tested?

Since the ``Filters`` and ``Properties`` translation elements are shared, I search for them in the code. 

``Filters`` does not seem to be used anywhere else than in the context menu where it does refer to item filters.

``Properties`` is also used for transitions properties and my change is not good in that context ; separate translation elements should be created for different contexts.
### Checklist:

- [ ] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] My code is not on the master branch.
- [ ] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.

### CODE REVIEW

**Comment by ysalmon** [comment]
Due to ideas.objproject.com being down at the moment, I paste here what I had the good ideao to copy before validating the form there :

I especially have experienced problems with duplication of Xcomposite sources : it seems to create problems when two sources grab the same window.

Some ideas :

1. The expression "Scene item" is used in places in the GUI but not everywhere. In this draft PR I made some change to make things more regular and explicit <https://github.com/obsproject/obs-studio/pull/2603>. What we want to display in the Sources panel are actually the Items, so the panel should bear that name.


2. There are some conundrums regarding names.
2.a. It still unclear to me if the names displayed in the Sources panel are the names of the Items or the name of their Sources. I can create two Color source items in the same scene with the same name, and they do share their source, but they are still two different items as they can be positioned differently. Do Sources have names ? They should, if we want to refer to things by name.

2.b. No two items should bear the same name, even if they share the same source.

2.c. A possible scheme would be to retain string names for sources and have a numerical discriminator for items. Eg. I could have a source named Background and two items (in the same or different scenes) automatically named Background#1 and Background#2.

2.d. Another way would be to let the user name the items at will with strings (but unique names) and give him or her the possibility to show the respective sources in a column of the Scene items panel.

3. Groups behave strangely
3.a. Can you confirm that it is semantically equivalent to copy an item then "Paste (reference)" it or to Create an new item and selecting the old one as existing source ?
3.b. If so, why is it that :
  - when creating a Group, it is not possible to use an existing group ;
  - copy and reference-pasting a Group is possible between different scenes ;
  - copy and reference-pasting a Group in the same scene is not grayed out but seems to do nothing ?

4. Being able to get, in a window similar to the Advanced audio properties, a list of all sources and in which scenes and as which items they are used would be another way of giving clarity. Furthermore, it would help managing sources that behave strangely when they are connected to the same input (Xcomposite, cameras).

5. To give the user insight about all this in the app, it would be very useful if on first run, one or several Example scene collections would be created. The concepts and features would be explained through self-referent Text items and sources. It could be made into a Tutorial with a visible Text item instructing the beginner to activate the visibility of another text item that would contain a next learning step.

**Comment by dodgepong** [comment]
> Due to ideas.objproject.com being down at the moment

It should be up, but that is the wrong URL. The correct URL is https://ideas.obsproject.com/

**Comment by kkartaltepe** [comment]
"The expression "Scene item" is used in places in the GUI", Where? It sounds more like reading the code which is not the usual confused users first step to understand how to use OBS.

**Comment by ysalmon** [comment]
Yes, I mistyped the URL here but the site really was down ; should we transfer to there ?

The context menu in the Sources panel has "Group Selected Items", and when deleting several items at the same time, the confirmation dialog uses the word "item" as well.

**Comment by Fenrirthviti** [comment]
I don't like the idea of changing to a poorly defined term that is barely used anywhere to explain functionality. I feel like this change will only confuse users, rather than help them.

I don't have a suggestion at this time on how to better communicate this, but this is also not exactly a common issue that comes up. I feel like we might be tackling the wrong problem here.

**Comment by kkartaltepe** [comment]
I agree, users understand these elements as sources. So to change the name would be hugely confusing not only to our English speaking users but to everyone who would have to re-translate and users who would have to re-interpret the translated change.

Changing the places noted where "Items" is used to "Sources" seems almost reasonable but im not sure there is really any benefit.

Exposing the internal implementation of scene items to users seems like a "bad idea". Source references are already exposed and relatively understood by the userbase via the "Add existing" functionality. The discoverability could certainly be improved in places that skip this prompt (like copy and paste). but I don't think forcing users to relearn core functional vocabulary is a good solution to that.

**Comment by ysalmon** [comment]
To clarify, this is a response to @WizardCM's comment on [this idea](https://ideas.obsproject.com/posts/790/add-a-deep-copy-operation-clarify-globality-of-objects-in-the-doc), itself prompted by his answer to my issue #2600. Originally, I proposed to make things more clear in the documentation, not necessarily in the app itself.

I am not familiar with the users' familiarity with vocabulary and the concrete proposal here is to illustrate the idea ; maybe it is better to keep Sources for what is in a scene and has a transformation, and to find another name for what can be connected to actual input devices and has properties and filters.

But it remains that currently the distinction between those two kinds of objects is very unclear in both the app and the documentation, and this is hugely confusing.

File: UI/data/locale/en-US.ini [hunk 1]
```diff
@@ -477,7 +477,7 @@ Basic.Filters="Filters"
 Basic.Filters.AsyncFilters="Audio/Video Filters"
 Basic.Filters.AudioFilters="Audio Filters"
 Basic.Filters.EffectFilters="Effect Filters"
-Basic.Filters.Title="Filters for '%1'"
+Basic.Filters.Title="Filters for the source of '%1'"
```
Comment (dodgepong) [inline UI/data/locale/en-US.ini:480]: Maybe `Filters for source '%1'`?

File: UI/data/locale/en-US.ini [hunk 2]
```diff
@@ -516,7 +516,7 @@ Basic.Main.AddSourceHelp.Text="You need to have at least 1 scene to add a source
 
 # basic mode main window
 Basic.Main.Scenes="Scenes"
-Basic.Main.Sources="Sources"
+Basic.Main.Sources="Items"
```
Comment (dodgepong) [inline UI/data/locale/en-US.ini:519]: I think the term "Source" is far too ingrained in people's mind that changing the title of the Sources box would be far too confusing for people, not to mention invalidate every OBS tutorial ever.

**Comment by PatTheMav** [comment]
Closing this after 3 years of inaction.

**Pull request closed without merging**